Dockerfile.dist

```diff
@@ -31,6 +31,7 @@ ARG TARGETOS
 ARG TARGETARCH
 
 ARG BUILD_LIST="darwin_amd64 darwin_arm64 linux_amd64 linux_arm64 linux_ppc64le linux_s390x windows_amd64"
+#ARG BUILD_LIST="linux_amd64"
```

Comment:
Handy for some kinds of local testing - you don't want to wait 40 minute for all the different binaries to build.